pkg/goenv-go-1.4.2/debian/copyright

```diff
@@ -0,0 +1,58 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: golang
+Source: https://go.googlecode.com/hg/
+
+Files: *
+Copyright: © 2012 The Go Authors. All rights reserved.
+License: Go
+
+Files: debian/*
+Copyright: 2015 Alex Tomlins <alex.tomlins@digital.cabinet-office.gov.uk>
```

Comment:
Should this be `Copyright (c) 2015 Crown Copyright (Government Digital Service)`?